uswds/uswds
# USWDS - POAM: December ‘24

# Summary

Updated vulnerable dependencies, updated indirect dependency for elliptic
## Breaking change

This is not a breaking change.

## Related issue

uswds/uswds-team#413

## Preview link

[Preview link:](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/cb-poam-dec-24/?path=/story/design-tokens-fonts--fonts)


## Major changes

Before:

```node
59 vulnerabilities (1 low, 31 moderate, 27 high)
```

After:

```node
55 vulnerabilities (0 low, 29 moderate, 26 high)
```

`
## Testing and review

1. Pull down and run `npm install`
2. Confirm no installation errors
3. Run `npx gulp buildSass` to test sass compilation
6. Run `npm run test`
8. Confirm no testing or linting errors 
9. Run `npm run build:html` and confirm the script runs as expected.
10. Start local or visit preview link
11. Confirm there are no visual regressions

## Dependency updates

| **Dependency name** | **Old version** | **New version** |
| --- | --- | --- |
| sass  | 1.81.0 | 1.83.0 |
| sass-embedded | 1.81.0 | 1.83.0 |
| snyk | 1.1294.1 | 1.1294.3 |
| stylelint | 16.10.0 | 16.11.0 |
| webpack | 5.96.1 | 5.97.1 |
